gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -0,0 +1,650 @@
+----
+
+** {empty}
++
+[source,kubectl]
+----
+kubectl -n gateway describe service
+----
+
+//TODO: this was from Israel. Not clear what upgrade error and 
+//      why we would uninstall the ingress for the namespace. 
+//Israel: "If you receive an upgrade error when execute the command, you can  
+//         remove the ingress released chart or omit the released argument:"
+//
+//## Command to remove ingress chart released
+//helm -n <namespace> uninstall ingress
+//
+// TODO: WHEN WOULD WE OMIT -i --create-namespace --wait?
+//## Command without install ingress chart arguments
+// helm -n <namespace> upgrade \
+//ingress flex-gateway/flex-gateway \
+//--set-file registration.content=<registration_file>
+
+[[check-cluster]]
+== Check Your Cluster
+
+Before deploying Flex Gateway to your Kubernetes cluster in <<deploy-flex>>, you must make sure that your cluster is ready. To check, you can use `kubectl` to list and select (use) a cluster and to get a list of nodes that the cluster uses. If any steps in this section fail, the deployment process is likely to produce errors.
+
+To prevent potential timeout and connectivity errors when running procedures in this guide, your cluster must meet the <<k8-cluster-reqs>>. 
+
+. Run `kubectl config get-contexts` to list the clusters, for example:
++
+[source,kubectl]
+----
+kubectl config get-contexts
+----
++
+This command is particularly helpful if you have multiple Kubernetes clusters. You can use the command to get the name of the the correct cluster. For example, assume that you have a variety of clusters (`my-cluster`, `my-aks`, `my-gke`, `my-eks`). The command prints a list of your clusters. 
++
+The following _example_ shows a list of clusters produced by the command:
++
+----
+CURRENT   NAME          CLUSTER       AUTHINFO    NAMESPACE
+          my-cluster    my-cluster                my-cluster1
+          my-aks        my-aks                    my-aks
+          my-gke        my-gke                    my-gke
+          my-eks        my-eks                    my-eks
+----
++
+If your cluster is not listed and you are using a custom cluster context file on your machine to connect to your cluster, you might need to add its path to a `KUBECONFIG` environment variable that points to that file. By default, `kubectl` searches for a file named `config` in the `$HOME/.kube` directory. That file lists each `context`, which provides properties such as the cluster name, user, and certificate authority data. 
++
+You can check by running `echo $KUBECONFIG` from your terminal window, which prints a environment variable's value, similar to this example: 
++
+----
+{KUBECONFIG}:/Users/me/.kube/config:/Users/me/.kube/additional-clusters/my-gke:/Users/me/.kube/additional-clusters/my-aks:/Users/me/.kube/additional-clusters/my-eks
+----
++
+Adding the path to any additional cluster context files on your machine enables the `kubectl config get-contexts` to list them and for you to use them.
```

Comment:
```suggestion
Adding the path to any additional cluster context files on your machine enables the `kubectl config get-contexts` command to list them and for you to use them.
```

Reply:
list-contexts to list the contexts

Reply:
enables... so that you can list and use the contexts?